theme/src/theme/AddToMetaMaskButton/index.tsx

```diff
@@ -42,7 +42,7 @@ export default function AddToMetaMaskButton() {
 
   return (
     <button
-      className={`button button--primary margin`}
+      className={`button button--primary button--lg margin-bottom--md`}
       onClick={() => addNetwork()}
     >
       Add ShimmerEVM Testnet to MetaMask
```

Comment:
```suggestion
      Add to MetaMask
```

Can we change it to this? Else the button overflows on mobile.

Reply:
Fine for me 👍🏻